src/BirthDate/BirthDate.jsx

```diff
@@ -2,12 +2,11 @@ import React from "react";
 import PropTypes from "prop-types";
 
 export default function BirthDate({ date, style }) {
-  if (date)
-    return (
-      <div style={style}>
-        {date.split("-")[2]}.{date.split("-")[1]}.{date.split("-")[0]}
-      </div>
-    );
+  return (
+    <div style={style}>
+      {date.split("-")[2]}.{date.split("-")[1]}.{date.split("-")[0]}
```

Comment:
will it panic in case of date is undefined?

Reply:
It'll complain in the proptypes since it is required.

Reply:
but shouldn't we handle it gracefully?

Reply:
Well we could do it, but I don't think its up to the component creator but rather the people using the component. Other wise all of our components will have {exist ? prop : handle gracefully} which doesn't really scale or work. I don't think its unreasonable to require a user to use a component properly.